zombienet/0004-webrtc/generate-warp-sync-database.toml

```diff
@@ -0,0 +1,17 @@
+# this file is not intended to be executed in CI stage
```

Comment:
If you are reusing database from `0001` you probably don't want to have it here. It is only used to (re-)generate database locally and update it on googlecloud storage.